### ISSUE TITLE
Cannot build and run the examples. “npm run build” fails

### ISSUE THREAD

**Issue by paulkoanui** [ory/editor#422]
Attempted to perform a fresh install per the readme instructions.  I am unable to get a successul build.

Here is the console output from `git clone` and `npm i`:
https://gist.github.com/paulkoanui/e50b7f9289a8a8b658050a803e493f9a 

Next, here is the console output from `npm run build`.  This step fails.
https://gist.github.com/paulkoanui/357e8b08e8b76ea384339e4141d43345 

It fails to compile the examples.  Any idea why a vanilla install like this wouldn’t work?

Thanks,
Paul

**Comment by arekkas** [comment]
Hi, sorry for the long response time. Could you try:

```
npm run clean
npm i
npm run build:lib
```

**Comment by paulkoanui** [comment]
Thanks for responding! 

I ran the commands that you provided and they worked.  Here's the full log:
https://gist.github.com/paulkoanui/740221bbbcbb4633287f5f5b9fb4f104 

I see that `build:lib` skips building the examples which is where my build was failing (`npm run build`).  

In readme.md, `npm run build` is the recommended way to build the project in order to run the examples.  That method of building the project still fails for me.  Here's that log:
https://gist.github.com/paulkoanui/d8f8446dd9eae0d1c27a62532e2b805f

Ultimately, I can build everything but the examples and that's ok for me.  However, in case anyone else is unable to build the project per the readme instructions, perhaps the second log I've attached will be helpful.

Thanks,
Paul
